sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -86,67 +84,98 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
     this.mapboxRenderer = mapboxRenderer
     this.renderHandlerThread = handlerThread
     this.eglCore = eglCore
+    this.eglSurface = eglCore.eglNoSurface
   }
 
-  private fun postPrepareRenderFrame() {
-    renderHandlerThread.post { prepareRenderFrame() }
+  private fun postPrepareRenderFrame(delayMillis: Long = 0L) {
+    renderHandlerThread.postDelayed(
+      {
+        prepareRenderFrame()
+      },
+      delayMillis
+    )
   }
 
-  private fun checkSurfaceReady(creatingSurface: Boolean): Boolean {
+  private fun isValidRenderingEnvironment(creatingSurface: Boolean): Boolean {
```

Comment:
Suggest to rename since it actually inits and not only checks.
`check` / `prepare` probably